src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -28,6 +34,7 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
     @Input() min?: dayjs.Dayjs; // Dates before this date are not selectable.
     @Input() max?: dayjs.Dayjs; // Dates after this date are not selectable.
     @Input() shouldDisplayTimeZoneWarning = true; // Displays a warning that the current time zone might differ from the participants'.
+    @Input() pickerType = DateTimePickerType.DEFAULT; // Only show the date picker, not the time picker.
```

Comment:
```suggestion
    @Input() pickerType = DateTimePickerType.DEFAULT; // Show the date and time picker
```
I think this comment is wrong, isn't it?